### ISSUE TITLE
Table component accessibility - extra column may cause confusion (AVT 3)

### ISSUE THREAD

**Issue by johnharan** [IBM/carbon-components-angular#297]
Two issues (which seem to be related to the same 'th' that's added to the right of the table).
1. JAWS reads 11 columns instead of 10 columns for the table.
2. The right extreme of the table has a visual defect at certain resolutions, visible in the below screenshot. Note how there is a gap between the end of the highlighted row and the right border.

![table](https://user-images.githubusercontent.com/8806230/49166285-ef135e00-f32a-11e8-921a-a12370bdc908.png)

> Is this a feature request (new component, new icon), a bug, or a general issue?

bug

> Is this issue related to a specific component?

Table component

> What browser are you working in?

Firefox 63.0.3 (64-bit)

> What version of the Carbon Design System are you using?

"carbon-components": "9.53.2",
"carbon-components-angular": "2.0.8",

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IWR

**Comment by johnharan** [comment]
@cal-smith @zvonimirfras Hi guys, I'm still seeing the issue mentioned above. IMO the border gap stands out and makes the table appear less polished  - is there any plan to address this? Thanks.

**Comment by cal-smith** [comment]
Yep, we've got this in our Q1 project so it'll get some eyes on it soon ...
